script/Ciel/Day 8/08_01_CIEL08_4_2.txt

```diff
@@ -96,67 +101,70 @@ Her outrageous outburst brings about an even more outrageous outburst in respons
 [sha:6b8ca1261bd6c484ad5e2356865513a5b06adfe0]{
 -- Page 419, Offset 19304.
 -- というか、明日もここで待ち合わせでいいんだな!?」
-I mean, we're meeting here again tomorrow, right?"
+Hold on, we're meeting here again tomorrow, right?"
 }
 [sha:a73be6540ad29f3513aaf1ad66c717ce2e8cc0e6]{
 -- Page 420, Offset 19305.
 -- 「あったりまえですぅー！
-"Obbbbb―――viously!
+"%{i}Obbbbb%{/i}―――viously!
```

Comment:
Uhh, shouldn't it be the Os that get extended instead of the Bs?

Reply:
Yeah, but that's just because you  exaggerate words in Japanese by using glottal stops, while in English you do it by extending vowels. Try saying 'Obbbbb―――viously!' out loud.

Reply:
I did! Github won't let me upload a sound file so [here](https://secure.xvjpf.org/obviously.mp3)

Reply:
Lol.

I don't think that way of saying it quite fits in this scene, though I think I see your perspective a bit better.